Source/Docs/usb-flash-drive-installation.rst

```diff
@@ -4,47 +4,86 @@ USB Flash Drive Installation
 
 While most of the development and testing of MOSA is done using virtualization software, MOSA does indeed boot on real hardware too.
 
-Below are the steps for deploying a MOSA disk image to a USB flash drive:
+There are several ways to put MOSA on a USB flash drive. Below are the most common ones, including the one we'll be following here:
 
-.. tip:: This page assumes you're using Windows, even though it's using the **dd** utility.
+- `Ventoy <https://ventoy.net/>`__ (All platforms)
+- `Rufus <https://rufus.ie/>`__ (Windows only)
+- dd (Linux, `unofficial Windows version <http://www.chrysocome.net/dd>`__ exists)
 
-1. Create a MOSA disk image using the MOSA Launcher Tool.
-
-2. Download the `dd <http://www.chrysocome.net/dd>`__ utility for Windows.
+We'll be following the ``dd`` way here.
 
-3. Copy the ``dd.exe`` executable to the build directory (usually a sub-folder under temp):
+Windows
+=======
 
-.. code-block:: text
+1. Create a MOSA disk image using the MOSA Launcher Tool.
 
-  %TEMP%\MOSA  
+2. Download the unofficial dd utility for Windows, as linked above.
 
-4. Open a command prompt window and change directory to the build directory.
+3. Open a command prompt window and change directory to the build directory (usually a subfolder under ``%TEMP%``).
 
 .. code-block:: text
 
   cd %TEMP%\MOSA 
 
+4. Copy the ``dd.exe`` executable to the current directory.
+
 5. Connect the USB flash drive you wish to ERASE and install the MOSA image onto.
 
 .. danger:: Data on the USB flash drive will be lost!
 
 6. Determine the device path for the USB flash drive.
 
-Get a list all the block devices on your system by typing the command below. Find the one for the USB flash drive you just connected. 
+Get a list all the block devices on your system by typing the command below. Find the one for the USB flash drive you just connected.
 
-.. danger:: Be careful! A mistype or wrong drive selection, may corrupt your hard drive or other storage devices. Do not proceed unless you completely understand these steps.
+.. danger:: Be careful! A mistype or wrong drive selection may corrupt your hard drive or other storage devices. Do not proceed unless you entirely understand these steps.
```

Comment:
Insert "are" before entirely.

Reply:
That would be incorrect: "you are entirely understand" does not make sense. Unless it's rewritten like so: "you are entirely on your own" for example.

Reply:
Correct; I misread from on my mobile device.